hs_core/management/commands/solr_query_json.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
From Alva: this can be deleted. It is part of a failed approach to speeding up SOLR responses.